### DISCUSSION TITLE
Create "new-table"

### DISCUSSION THREAD

**Discussion by boheme61** [hassio-addons/addon-sqlite-web#191] [Q&A]
Hi there, im pretty new to HA, and trying to create a "new table" in the sqllite-add-on, by clicking on the [Create] "new-table-name" ... to import CSV content into manually , but keep getting (error 404: Not Found) , am i missing something (i.e. do i have to create a file in /config first ?)
BR Jörgen

**Comment by boheme61** [comment]
is it not possible to create "new-table" in "home-assistant_v2.db" via sqlite-addon ?, maybe i should add, that i have installed Grafana and InfluxDB as well, and everything´s running fine ... and cant seems to find where that 404 is logged, so no help there
/Jörgen

**Reply by boheme61** [reply]
adding pic, from tra
![create_table](https://user-images.githubusercontent.com/37634316/143661567-2b166872-0b6b-4a80-a387-46c8fee9ac12.jpg)
ce

**Reply by boheme61** [reply]
another from ha_logger

![ha_logger](https://user-images.githubusercontent.com/37634316/143662070-55398cc7-672a-4d2b-802d-fb6745ae8337.jpg)

**Reply by hitnrun30** [reply]
Did you ever find an answer?

**Comment by boheme61** [comment]
To be honest i dont remember, thou i do recall that i managed to create a table, but never got to the point of importing from external CSV file, but if you search the Homeassistant-forum i think you would be able to find a solution for that. Im Sorry but i dont currently have access to my Homeassistant , and havent been in the forum past 2 month, so i cant give you a better " hint" or currently be able to assist you further ... but i know their are several solutions to import data from CSV to homeassistant, depending upon your DB setup, and the the way you want "present" this information in " frontend"
